Neos.Flow/Classes/I18n/AbstractXmlParser.php

```diff
@@ -55,7 +55,7 @@ protected function getRootNode(string $sourcePath): \SimpleXMLElement
             throw new Exception\InvalidXmlFileException('The path "' . $sourcePath . '" does not point to an existing and accessible XML file.', 1328879703);
         }
         libxml_use_internal_errors(true);
-        $rootXmlNode = simplexml_load_file($sourcePath, 'SimpleXmlElement', \LIBXML_NOWARNING);
+        $rootXmlNode = simplexml_load_string(file_get_contents($sourcePath), 'SimpleXmlElement', \LIBXML_NOWARNING);
```

Comment:
Maybe add link to the simplexml bug to document why `load_file` isn't used?

Reply:
Isn't that what `git blame` is for? 😬